tests/framework/extensions/namespaces/namespaces.go

```diff
@@ -142,3 +141,43 @@ func CreateNamespace(client *rancher.Client, namespaceName, containerDefaultReso
 	}
 	return newNamespace, nil
 }
+
+
+// ListNamespace is a helper function that uses the dynamic client to get a list of all namespaces from a cluster from all projects.
+func ListNamespace(client *rancher.Client, clusterID string) (*kubeUnstructured.UnstructuredList, error) {
```

Comment:
Anu have this function convert the unstructured list to a namespace list. There are example in the other extensions how to do this. We don't want to return these functions to return unstructureds. But actually Ugur has a PR for this, that we'll get merged in first https://github.com/rancher/rancher/pull/38171